docs/contributor/code-contribute.md

```diff
@@ -346,8 +347,18 @@ Make sure that the title for your pull request uses the same format as the subje
 ### Pass all the CI checks
 
 Before merge, All test CI should pass green.
-The `codecov/project` should also pass. This means the coverage should not drop.
-
+- The `codecov/project` should also pass. This means the coverage should not drop.
+- The `DCO` should also pass.  It requires all commit messages to contain the `Signed-off-by` line with an email address that matches the commit author.
+  - Contributors sign-off that they adhere to these requirements by adding a `Signed-off-by` line to commit messages.
+    ```shell
+    This is my commit message
+    
+    Signed-off-by: Random Developer <random@developer.example.org>
+    ```
+  - Git even has a `-s` command line option to append this automatically to your commit message:
+    ```shell
+     git commit -s -m 'This is my commit message'
+    ```
```

Comment:
```suggestion
  - Git provides a convenient flag `-s` in your commit command to sign-off automatically:
    ```shell
     git commit -s -m 'This is my commit message'
    ```
    - Contributors can also sign-off manually by adding a `Signed-off-by` line to commit messages as the following format, make sure the email matches your github account or the check bot won't pass.
    ```shell
    This is my commit message
    
    Signed-off-by: Random Developer <random@developer.example.org>
    ```
```